qid-batch-runner.yml

```diff
@@ -112,7 +125,7 @@ spec:
         - name: cloud-sql-certs
           secret:
             secretName: cloud-sql-certs
-            defaultMode: 0400
+            defaultMode: 0660
```

Comment:
Does this need to be `0600` instead?

Reply:
I believe it worked with either. Think it ended up at 0660 because we were playing around with the permissions but you are right, for clarity it should be 0600. Have changed it now.